services/QuillLMS/app/models/concept_result.rb

```diff
@@ -70,6 +70,7 @@ def self.init_from_json(data_hash)
       concept_id: data_hash[:concept_id],
       attempt_number: metadata[:attemptNumber],
       correct: ActiveModel::Type::Boolean.new.cast(metadata[:correct]),
+      old_concept_result_id: data_hash[:old_concept_result_id],
```

Comment:
I'm trying to make sense of this addition.  Do concept_results (i.e. the newer ones) from the large migration last week last lack this value?  Or were they added somewhere else.

Reply:
Good question!  Actually, the migration took place entirely outside of the class structure (well, not totally, it did use `parse_extra_metadata`) in order to allow for bulk inserts.  You can see that code original migration code (which has already been replaced as I worked through the edge cases in the migration) here: https://github.com/empirical-org/Empirical-Core/blob/dd7ea7d7557/services/QuillLMS/app/workers/copy_old_concept_results_to_concept_results_worker.rb